CODEOWNERS

```diff
@@ -35,8 +35,8 @@ extensions/filters/common/original_src @klarose @mattklein123
 /*/extensions/filters/http/jwt_authn @taoxuy @lizan @tyxia @yanavlasov
 # grpc_field_extraction http filter extension
 /*/extensions/filters/http/grpc_field_extraction @taoxuy @nareddyt @yanavlasov
-# proto_message_logging http filter extension
-/*/extensions/filters/http/proto_message_logging @dchakarwarti @taoxuy @yanavlasov
+# proto_message_scrubbing http filter extension
```

Comment:
I thought our plan is to use this for versioning/filtering scrubbing as well? @dchakarwarti  Sumit should have context.

Reply:
I confirmed with Sumit that the Proto Scrubber filter for visibility-related changes is separate from the PML/PMS filter and will be developed independently.

My rationale for renaming **Proto Message Logging** was that we aren't actually logging but rather writing extracted fields into dynamic metadata. However, I understand that the name **Proto Message Scrubbing** might be confusing.

Please let me know if you prefer to proceed with the `Proto Message Logging` filter as it is.

Reply:
I see. Then Proto Message Extraction might be better? But would it confuse with gRPC field extraction? Proto Message Logging is indeed confusing too because it's not logging.

Reply:
Didn't we think about merging this to gRPC field extraction filter?

Reply:
> Didn't we think about merging this to gRPC field extraction filter?

Haven't we considered this already[1] and decide not to merge?

https://docs.google.com/document/d/1K7ANyvLkolM5o_pQP2hWpY0NJfI9rQ1czO1e5YaYzmo/edit?tab=t.0#heading=h.282wm8yq0rz6

Reply:
In that case, if we are looking for alternate names, how about:

1. Field Metadata Extractor
2. Proto Metadata Extractor
3. Proto Field Metadata Filter
4. Proto Field Capture

`Proto Metadata Extractor` might be the best balance between specificity and clarity.

I am okay if we decide to go with `Proto Message Extraction`.

Please let me know if any one sounds good.

Reply:
I see. It's too long ago and I forgot about it. Then either way sounds good to me.

Reply:
SG on Proto Message Extraction

Reply:
I have renamed the filter to `Proto Message Extraction` filter and thereby renamed necessary files and variables.

Thank you!